model/error.go

```diff
@@ -2,10 +2,16 @@ package model
 
 import "fmt"
 
+// ErrorKey is error for key api
+type ErrorKey struct {
+	KeyName string `json:"key_name,omitempty"`
+}
+
 // Error is an API error.
 type Error struct {
-	Code    int    `json:"code,omitempty"`
-	Message string `json:"message,omitempty"`
+	Code    int      `json:"code,omitempty"`
+	Message string   `json:"message,omitempty"`
+	Key     ErrorKey `json:"key,omitempty"`
```

Comment:
I don't see this in the documentation: https://lokalise.co/api2docs/curl/#resource-errors

Have you identified this against the API? In that case it might be missing.

Reply:
updated, move errorKeys into model/key.go
it's different from api return.
and should be include in KeysResponse

Reply:
Some API endpoints with bulk actions can contain `errors` array in the response. Such as Screenshots::Add, Languages::Add, Keys::Add, Keys::BulkUpdate. Each error contain `code`, `message` and some identifier.

Errors example for Languages::Add
```json
{
   "errors": [
      {
         "message": "Language is already added to project",
         "code": 208,
         "lang_iso": "en"
      },
      {
         "message": "Language is already added to project",
         "code": 208,
         "lang_iso": "lv"
      }
   ]
}
```
In the example the identifier is "lang_iso", the only uniq parameter from requests body.

For now, those identifiers are:
Screenshots::Add -- object "screenshot" (from requests body)
Languages::Add -- string "lang_iso"
Keys::Add -- object "key_name" `{"ios": {name}, "android": {name}, "web": {name}, "other": {name}}`
Keys::BulkUpdate -- integer "key_id"

Not sure, if it is required to be documented. Those identifiers are used only for tracing what has went wrong. They are not strict structured also. We may add new errors list to other endpoints later as well (Contributors::Add, Comments::Add etc).